COLLABORATOR_GUIDE.md

```diff
@@ -224,23 +237,45 @@ Save the file and close the editor. You'll be asked to enter a new
 commit message for that commit. This is a good moment to fix incorrect
 commit logs, ensure that they are properly formatted, and add
 `Reviewed-By` lines.
+* The commit message text must conform to the [commit message guidelines](../CONTRIBUTING.md#step-3-commit).
 
 Time to push it:
 
 ```text
 $ git push origin master
 ```
+* Optional: Force push the amended commit to the branch you used to
+open the pull request. If your branch is called `bugfix`, then the
+command would be `git push --force-with-lease origin master:bugfix`.
+When the pull request is closed, this will cause the pull request to
+show the purple merged status rather than the red closed status that is
+usually used for pull requests that weren't merged. Only do this when
+landing your own contributions.
+
+* Close the pull request with a "Landed in `<commit hash>`" comment. If
+your pull request shows the purple merged status then you should still 
+ add the "Landed in <commit hash>..<commit hash>" comment if you added
+ multiple commits.
+
+* `./configure && make -j8 test` (`-j8` builds node in parallel with 8
+  threads. Adjust to the number of cores (or processor-level threads)
+  your processor has (or slightly more) for best results.)
```

Comment:
I think this is an excessive number of brackets. 

(`-j8` builds node in parallel with 8 threads. Adjust to the number of processor cores/threads your machine has for best results.)